### PR TITLE
Feature/sg 1085 docs for image dups

### PR DESCRIPTION
**Pull request by shaydeci**

### CODE REVIEW

File: documentation/image_duplicates.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+# Tutorial: Finding Image Duplicates using the ImageDuplicates Feature Extractor
+In this tutorial, we will explore how to utilize the ImageDuplicates class to identify duplicate images within a specified directory containing images. The ImageDuplicates class is a feature extractor that employs Difference Hashing to detect duplicate images based on their hash codes. We will guide you through the process of initializing and using this class to find duplicates within the COCO2017 detection dataset and examine the results.
+
+
+## Step 1: Adding the ImageDuplicates to the Image Features Report Section Features List
+Just like any other feature extractor, you need to include it in your feature list within the appropriate report section.
+Since finding duplicates is not task-specific, it belongs in the "Image Features" report section. In your detection.yaml configuration file, add it as follows:
```
Comment (Louis-Dupont) [inline documentation/image_duplicates.md:7]: I would state that this feature is the only that is not in the default features because it requires explicit `train_image_dir`, while other features dont require to change the `.yaml`

File: documentation/image_duplicates.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
+# Tutorial: Finding Image Duplicates using the ImageDuplicates Feature Extractor
+In this tutorial, we will explore how to utilize the ImageDuplicates class to identify duplicate images within a specified directory containing images. The ImageDuplicates class is a feature extractor that employs Difference Hashing to detect duplicate images based on their hash codes. We will guide you through the process of initializing and using this class to find duplicates within the COCO2017 detection dataset and examine the results.
+
+
+## Step 1: Adding the ImageDuplicates to the Image Features Report Section Features List
```
Comment (Louis-Dupont) [inline documentation/image_duplicates.md:5]: To keep style constant with other `.md`, I think it's better to write `## 1. Adding ...` instead of `## Step1: Adding...`

File: documentation/image_duplicates.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+Now that we have added ImageDuplicates to our feature list, running the analysis is straightforward. Here's an example:
+````python
+
+# Note: This example will require you to install the super-gradients package
+from super_gradients.training.dataloaders.dataloaders import coco2017_train, coco2017_val
+from data_gradients.managers.detection_manager import DetectionAnalysisManager
+
+
+if __name__ == "__main__":
```
Comment (Louis-Dupont) [inline documentation/image_duplicates.md:39]: Let's drop the `__name__` part, it will look simpler

File: documentation/image_duplicates.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
+# Tutorial: Finding Image Duplicates using the ImageDuplicates Feature Extractor
+In this tutorial, we will explore how to utilize the ImageDuplicates class to identify duplicate images within a specified directory containing images. The ImageDuplicates class is a feature extractor that employs Difference Hashing to detect duplicate images based on their hash codes. We will guide you through the process of initializing and using this class to find duplicates within the COCO2017 detection dataset and examine the results.
+
+
```
[the remark this answers is not in the document]
Comment (shaydeci) [reply documentation/image_duplicates.md:4]: I think this doesn't really make sense here. Its a short sequential tutorial.

File: documentation/image_duplicates.md (hunk 1 above)
Comment (shaydeci) [reply documentation/image_duplicates.md:7]: Added in latest commit.

File: documentation/image_duplicates.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+Please note that train_image_dir and valid_image_dir should point to the directories containing all the training and validation images,
+respectively. It's essential to understand that this feature extractor does not account for any internal dataset modifications, such as image alterations or exclusions.
+It operates solely on the standard loading of images within those directories without preprocessing.
+
+# Step 2: Running Analysis
+
+Now that we have added ImageDuplicates to our feature list, running the analysis is straightforward. Here's an example:
+````python
+
+# Note: This example will require you to install the super-gradients package
```
[the remark this answers is not in the document]
Comment (shaydeci) [reply documentation/image_duplicates.md]: Added in latest.

File: documentation/image_duplicates.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+
+```
+We have finished evaluating your dataset!
+
+The results can be seen in:
+    - /path/to/data-gradients/examples/logs/COCO2017DGAnalysis
+    - /path/to/data-gradients/examples/logs/COCO2017DGAnalysis/archive_20230905-141929
+```
+
+#Step 3: Examine Results
```
[the remark this answers is not in the document]
Comment (shaydeci) [reply documentation/image_duplicates.md]: Fixed

File: documentation/image_duplicates.md [hunk 7]
```diff
@@ -163,4 +165,22 @@ Lastly, let's explore the intersection of the train and validation sets:
...
+
+`/data/coco/images/val2017/000000140556.jpg`:
+![subsection](assets/000000080010.jpg)
+
+`/data/coco/images/train2017/000000535889.jpg`:
+
+![subsection](assets/000000535889.jpg)
+
+`/data/coco/images/val2017/000000465129.jpg`:
+![subsection](assets/000000465129.jpg)
```
Comment (Louis-Dupont) [inline documentation/image_duplicates.md:182]: Sometimes the line is skipped and sometimes it is not. It eventually leads to a weird display (check when visualizing the .md)
Maybe it would look nicer with a table
```
| Image A | Image B |
|---------|---------|
| `/data/coco/images/train2017/000000080010.jpg`<br>![subsection](assets/000000080010.jpg) | `/data/coco/images/val2017/000000140556.jpg`<br>![subsection](assets/000000080010.jpg) |
| `/data/coco/images/train2017/000000535889.jpg`<br>![subsection](assets/000000535889.jpg) | `/data/coco/images/val2017/000000465129.jpg`<br>![subsection](assets/000000465129.jpg) |

```

**Review by Louis-Dupont — APPROVED** [review]
If you can try with the table (see other comment) but other than that LGTM

**Review by Louis-Dupont — APPROVED** [review]
LGTM
